### DISCUSSION TITLE
Preview Error

### DISCUSSION THREAD

**Discussion by tiffanysprague** [thegetty/quire#75] [Get Help] [answered]
Hi Quire folks,

Receiving a preview error in Terminal when trying to preview a repository that I've cloned to my local directory (created by another member of my team):

Error: spawn node_modules/.bin/webpack ENOENT

Not sure what this means... Any help is appreciated!

Thanks,
Tiffany

[preview error.pdf](https://github.com/thegetty/quire/files/6190848/preview.error.pdf)

**Comment by Erin-Cecele** [comment, marked as answer]
Hi @tiffanysprague, 

After you cloned the repo, did you run `quire install` prior to running `quire preview`? Here is the relevant section of the documentation: https://quire.getty.edu/documentation/getting-started/#copy-an-existing-project. 

Let me know if that helps,

Erin

**Reply by tiffanysprague** [reply]
Yup, that worked! Thanks, Erin—you're the best!!
Tiffany

**Reply by Erin-Cecele** [reply]
Wonderful! Happy to help, Tiffany!
